.github/workflows/masterbot.yml

```diff
@@ -3,11 +3,16 @@ on:
   push:
     branches:
       - master
-    tags-ignore:
-      - "**"
 jobs:
+  allow-workflow:
+    name: "Allow workflow"
+    if: "!startsWith(github.event.head_commit.message, '[Version Bump]')"
```

Comment:
Unfortunately, we can't put an `if` condition on an entire workflow, and there's no way to target commit messages otherwise.